chef-server-stats/folsom-stats

```diff
@@ -7,12 +7,15 @@
 fetch_metric({MetricKey, [MetricType = {type, histogram} ]}) ->
     {MetricKey, rpc:call(?ERCHEF, folsom_metrics,get_histogram_statistics, [MetricKey]), MetricType};
```

Comment:
Actually, it never worked on EC11, this is 12 only afaik. But I meant 12.0... for which this code was enabled, but also didn't work for similar reasons...